### PR TITLE
Add 3.12 headline features to release notes

### PR DESCRIPTION
**Pull request by ColeHiggins2**
Adding headline features and deprecations for release notes for rc2

### CODE REVIEW

File: _includes/manuals/3.12/1.2_release_notes.md [hunk 1]
```diff
@@ -4,10 +4,39 @@ This section will be updated prior to the next release.
...
+
+Foreman 3.12 now fully supports Puppet 8. While Foreman 3.11 unofficially supported Puppet 8, Puppet’s official puppetserver RPMs are known to break on Java. The installer now handles this for users. Users are encouraged to upgrade since Puppet 7 is expected to go end of life in February 2025.
+
+For more details, see https://projects.theforeman.org/issues/37686.
+
 ### Upgrade warnings
 
 ### Deprecations
 
+#### Port 8443 disabled by default
```
Comment (ekohl) [inline _includes/manuals/3.12/1.2_release_notes.md]: This is a Katello feature that doesn't affect regular Foreman. Please drop this and instead add the EL 8 deprecation.

**Review by ekohl — CHANGES_REQUESTED** [review]

File: _includes/manuals/3.12/1.2_release_notes.md (hunk 1 above)
Comment (ColeHiggins2) [reply _includes/manuals/3.12/1.2_release_notes.md]: Ack, Am I missing any other headline features, warnings, or deprecations? Aside from el 8

File: _includes/manuals/3.12/1.2_release_notes.md [hunk 2]
```diff
@@ -4,16 +4,54 @@ This section will be updated prior to the next release.
...
+
+Foreman 3.12 now fully supports Puppet 8. While Foreman 3.11 unofficially supported Puppet 8, Puppet’s official puppetserver RPMs are known to break on Java. The installer now handles this for users. Users are encouraged to upgrade since Puppet 7 is expected to go end of life in February 2025.
+
+For more details, see https://projects.theforeman.org/issues/37686.
+
 ### Upgrade warnings
 
 ### Deprecations
 
+#### Running Foreman on {EL} 8 removal in Foreman 3.13
```
Comment (ekohl) [inline _includes/manuals/3.12/1.2_release_notes.md]: `{EL}` is an asciidoc macro we can't use here
```suggestion
#### Running Foreman on Enterprise Linux 8 removal in Foreman 3.13
```

File: _includes/manuals/3.12/1.2_release_notes.md [hunk 3]
```diff
@@ -4,16 +4,54 @@ This section will be updated prior to the next release.
...
+
+Running on {EL} 8 is deprecated since Foreman 3.11.
+Foreman 3.13 will drop this support so users are encouraged to plan their {UpgradingDocURL}upgrading_EL_on_foreman_or_proxy_upgrading-connected[upgrade].
```
Comment (ekohl) [inline _includes/manuals/3.12/1.2_release_notes.md]: `{UpgradingDocURL}` is also a macro that needs to be expanded here. You're also copying the asciidoc URL style that doesn't work with markdown.

**Review by ekohl — CHANGES_REQUESTED** [review]

**Pull request merged**